app/helpers/root_helper.rb

```diff
@@ -1,9 +1,13 @@
 module RootHelper
-  def application_name
-    if @application.present?
-      @application.name
+  def gds_only_application_and_non_gds_user?
+    @application.gds_only? && !current_user.belongs_to_gds?
+  end
+
+  def title
+    if @application.blank? || gds_only_application_and_non_gds_user?
+      "You don’t have permission to use this app."
     else
-      "this application"
+      "You don’t have permission to sign in to " + @application.name + "."
```

Comment:
`in to #{@application.name}."` might be neater and more consistent .

Reply:
👍